tests/integration/graviton_test.py

```diff
@@ -282,9 +273,6 @@ def test_stable_teal_generation(subr, mode):
     oldfac: {
         "inputs": [(i,) for i in range(25)],
         "assertions": {
-            DRProp.cost: lambda args, actual: (
```

Comment:
In several cases, I completely removed assertions. I'm confident that we're good-to-go though because of the new "identities" tests.